tests/e2e-leg-11/client-proxy/setup-vdb/base/setup-vdb.yaml

```diff
@@ -25,6 +25,7 @@ spec:
   communal: {}
   local:
     requestSize: 250Mi
+  imagePullPolicy: IfNotPresent
```

Comment:
This cannot resolve image pulling issue if the customer uses "always" pull policy.

Reply:
Thanks for the help. Adding docker-push-vproxy into Makefile and updating the workflows inputs fixed it.